src/test/java/org/folio/logintest/LogEventsApiTest.java

```diff
@@ -89,26 +84,17 @@ public static void setUpClass(final TestContext context) {
 
     DeploymentOptions restDeploymentOptions = new DeploymentOptions()
       .setConfig(new JsonObject().put(HTTP_PORT, port));
-    vertx.deployVerticle(RestVerticle.class.getName(), restDeploymentOptions, res -> {
+    vertx.deployVerticle(RestVerticle.class.getName(), restDeploymentOptions)
+        .onComplete(context.asyncAssertSuccess(res -> {
       TenantAttributes ta = new TenantAttributes().withModuleTo("mod-login-1.1.0");
-      TenantAPI tenantAPI = new TenantRefAPI();
-      Map<String, String> okapiHeaders = Map.of("x-okapi-url", "http://localhost:" + port,
-          "x-okapi-tenant", TENANT_ID);
-      tenantAPI.postTenantSync(ta, okapiHeaders, handler -> async.complete(),
-          vertx.getOrCreateContext());
-    });
+      TestUtil.postSync(ta, TENANT_ID, port, vertx).onComplete(context.asyncAssertSuccess());
+    }));
```

Comment:
```
    DeploymentOptions restDeploymentOptions = new DeploymentOptions()
      .setConfig(new JsonObject().put(HTTP_PORT, port));
    TenantAttributes ta = new TenantAttributes().withModuleTo("mod-login-1.1.0");
    vertx.deployVerticle(RestVerticle.class.getName(), restDeploymentOptions)
        .compose(x -> TestUtil.postSync(ta, TENANT_ID, port, vertx))
        .onComplete(context.asyncAssertSuccess());
```

Reply:
Oh..Missed more of these.. Will do them all .